Changes between KDE 3.5.10 tag as modified by kdepim-3.5-patchset-05.tar.bz2
and KDE 3.5 branch r886802. Only looking at subdirectory kdepim/kabc here.
Logs since r857205 of the branch.

r857667 | scripty
SVN_SILENT made messages (.desktop file)
  M kabcdistlistupdater/kabcdistlistupdater.desktop
r859339 | scripty
SVN_SILENT made messages (.desktop file)
  M kabcdistlistupdater/kabcdistlistupdater.desktop
r861733 | scripty
SVN_SILENT made messages (.desktop file)
  M kabcdistlistupdater/kabcdistlistupdater.desktop
r862838 | scripty
SVN_SILENT made messages (.desktop file)
  M kabcdistlistupdater/kabcdistlistupdater.desktop
r866496 | scripty
SVN_SILENT made messages (.desktop file)
  M kabcdistlistupdater/kabcdistlistupdater.desktop
r869452 | scripty
SVN_SILENT made messages (.desktop file)
  M kabcdistlistupdater/kabcdistlistupdater.desktop
r875659 | scripty
SVN_SILENT made messages (.desktop file)
  M kabcdistlistupdater/kabcdistlistupdater.desktop
r886802 | scripty
SVN_SILENT made messages (.desktop file)
  M kabcdistlistupdater/kabcdistlistupdater.desktop

diff -u kdepim/kabc/kabcdistlistupdater/kabcdistlistupdater.desktop kdepim/kabc/kabcdistlistupdater/kabcdistlistupdater.desktop
--- kdepim/kabc/kabcdistlistupdater/kabcdistlistupdater.desktop	(revision 857205)
+++ kdepim/kabc/kabcdistlistupdater/kabcdistlistupdater.desktop	 (.../branches/KDE/3.5)	(revision 886802)
@@ -1,10 +1,20 @@
 # KDE Config File
 [Desktop Entry]
 Name=kabcdistlistupdater
+Name[nl]=kabcdistlijstbijwerker
 Exec=kabcdistlistupdater --disable-autostart
 Icon=misc
 Type=Application
 Comment=Tool to update the old distribution lists to the new ones.
+Comment[ca]=Eina per actualitzar les llistes de distribució antigues a les noves.
+Comment[es]=Herramienta para actualizar las antiguas listas de distribución al nuevo formato.
+Comment[et]=Tööriist vanade postiloendite uuendamiseks uue vormi kohaselt.
+Comment[nds]=Warktüüch, dat ole Verdeellisten to niege opfrischt
+Comment[nl]=Hulpmiddel om de oude distributielijsten bij te werken tot de nieuwe.
+Comment[sk]=Nástroj na aktualizáciu starých ditribučných zoznamov na nové.
+Comment[sr]=Алат за ажурирање старих дистрибуционих листи новима.
+Comment[sr@Latn]=Алат за ажурирање старих дистрибуционих листи новима.
+Comment[sv]=Verktyg för att uppdatera gamla distributionslistor till nya.
 Terminal=false
 NoDisplay=true
 X-KDE-autostart-condition=kabcdistlistupdaterrc:Startup:EnableAutostart:true
